plugins/vpp/ifplugin/descriptor/interface.go

```diff
@@ -508,6 +522,13 @@ func (d *InterfaceDescriptor) Validate(key string, intf *interfaces.Interface) e
 			}
 
 		}
+	case interfaces.Interface_GTPU_TUNNEL:
+		if intf.GetGtpu().SrcAddr == "" {
```

Comment:
VXLAN branch above parses IP addresses and returns `InvalidValueError` if parse fails. But we merely compare with an empty string. Shouldn't we validate if the string is a valid IP address?